GitHub.Collectors.Tests/Web/GitHubRateLimiterTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.CloudMine.Core.Collectors.Authentication;
+using Microsoft.CloudMine.Core.Collectors.Cache;
+using Microsoft.CloudMine.Core.Collectors.Telemetry;
+using Microsoft.CloudMine.Core.Collectors.Tests.Telemetry;
+using Microsoft.CloudMine.Core.Collectors.Tests.Web;
+using Microsoft.CloudMine.Core.Collectors.Web;
+using Microsoft.CloudMine.GitHub.Collectors.Tests.Helpers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.CloudMine.GitHub.Collectors.Web.Tests
+{
+
+    public class GitHubRateLimiterTestClass : GitHubRateLimiter
```

Comment:
What is the advantage of this class? It looks like the only method it overrides, it is simply calling another method from base? Is there a reason not to call that method directly (could be a visibility thing, let me know)